subprojects/docs/src/samples/buildCache/configure-by-init-script/init.gradle

```diff
@@ -0,0 +1,9 @@
+gradle.settingsEvaluated { settings ->
+    settings.buildCache {
+        // vvv Your custom configuration goes here
+        remote(HttpBuildCache) {
+            url = 'https://example.com:8123/cache/'
```

Comment:
This doesn't cause tests to fail because all the cacheable tasks have no sources, right?

Reply:
Yeah, we are not trying to pull something from the cache, we only want to see if the configuration DSL can be executed.